backend/core/domains/payments/models.py

```
@@ -1,27 +1,195 @@
+        related_name='primary_for_global_settings',
+        help_text="Primary payment gateway (pre-selected by default)"
+    )
+
+    class Meta:
+        verbose_name = "Payment Settings"
+        verbose_name_plural = "Payment Settings"
+
+    def clean(self):
+        """Validate that only one settings instance exists"""
+        if not self.pk and PaymentSettings.objects.exists():
+            raise ValidationError("Only one PaymentSettings instance is allowed.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)